fronts-client/src/components/inputs/InputRadio.tsx

```diff
@@ -95,18 +122,28 @@ export default ({
 	id,
 	dataTestId,
 	input: { ...inputRest },
+	checked,
+	icon = undefined,
+	usesBlockStyling = false,
 	...rest
 }: Props) => (
 	<>
 		<InputContainer data-testid={dataTestId}>
-			<RadioButtonContainer>
+			<RadioButtonContainer usesBlockStyling={usesBlockStyling}>
 				<Switch>
-					<RadioButton type="radio" {...inputRest} {...rest} id={id} />
+					<RadioButton
+						type="radio"
+						{...inputRest}
+						checked={checked}
```

Comment:
`checked` needs to be supplied directly, the spread operator can drop this value. The value is needed to manually drive the radio button, which doesn't play well with Redux forms (https://github.com/redux-form/redux-form/issues/124).